tests/cli/test_datasource_pandas.py

```diff
@@ -120,13 +117,10 @@ def test_cli_datasource_new(
     mock_emit,
     caplog,
     monkeypatch,
-    empty_data_context,
+    empty_data_context_stats_enabled,
```

Comment:
tests now explicitly enable `usage_statistics` by using the `_stats_enabled` fixture directly